File: java-junit/src/main/java/Game.java
public class Game {

    public boolean rules(int neighbors, boolean live) {
        if (neighbors < 2 || neighbors >= 4) {
            return false;
        }
        if (live == false) {
            if (neighbors == 3) {
                return true;
            }
            return false;


        }
        return true;
    }
}

File: csharp/kata/Game.cs
namespace kata
{
    public class Game
    {
        public static int Neighbors(Board board, int x, int y)
        {
            int live = 0;
            for (int i = 0; i < board.MaxX; i++)
            {
                for (int j = 0; j < board.MaxY; j++)
                {
                    if (board[i, j] && !(i == x && j == y)) live++;
                }
            }
            return live;
        }

        public static bool IsAlive(int neighbors, bool cellstate)
        {
            switch (neighbors)
            {
                case 0:
                case 1:
                    return false;
                default:
                    return true;
            }
        }
    }
}